save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesSourceService.kt

```diff
@@ -92,6 +92,17 @@ class TestSuitesSourceService(
             testSuitesSourceRepository.save(entity.toTestSuiteSource(organization, git))
         }
 
+    /**
+     * Raw update
+     *
+     * @param entity [TestSuitesSource] to be updated
+     * @return updated [TestSuitesSource]
+     */
+    fun update(entity: TestSuitesSource): TestSuitesSource {
+        entity.requiredId()
```

Comment:
what is it? if you added it to check that this entity is for update -- need to update error message and add a comment at least

Reply:
I like a message that is inside `requiredId()`
Will add comment

Reply:
Done